common/src/primitives/amount.rs

```diff
@@ -131,6 +131,12 @@ impl Amount {
     }
 }
 
+impl From<Amount> for u128 {
+    fn from(amount: Amount) -> Self {
```

Comment:
Please remove this and use the method `into_atoms()`

Reply:
Done.